pmm-app/tests/verifyQAN_test.js

```diff
@@ -121,3 +122,99 @@ Scenario('Verify Main Metric change reflects in URL @not-pr-pipeline', async (I,
   qanPage.changeMetricTo(metricToReplace, newMetricName);
   qanPage.verifyURLContains(qanPage.urlParts.lockTime);
 });
+
+xScenario(
+  'PMM-T175 - Verify user is able to apply filter that has dots in label @not-pr-pipeline',
+  async (I, qanPage) => {
+    const serviceName = 'ps_5.7_0.0.0.0_1';
+    qanPage.waitForNewQANPageLoaded();
+    const countBefore = await qanPage.getCountOfItems();
+    qanPage.applyFilterNewQAN(serviceName);
+    I.seeInCurrentUrl('service_name=' + serviceName);
+    I.waitForInvisible(qanPage.fields.newQANSpinnerLocator, 30);
+    const countAfter = await qanPage.getCountOfItems();
+    qanPage.verifyChangedCount(countBefore, countAfter);
+  }
+);
+
+xScenario(
+  'PMM-T172 - Verify that selecting a filter updates the table data and URL  @not-pr-pipeline',
+  async (I, qanPage) => {
+    const environmentName = 'ps-dev';
+    qanPage.waitForNewQANPageLoaded();
+    const countBefore = await qanPage.getCountOfItems();
+    qanPage.applyFilterNewQAN(environmentName);
+    I.seeInCurrentUrl('environment=' + environmentName);
+    I.waitForInvisible(qanPage.fields.newQANSpinnerLocator, 30);
+    const countAfter = await qanPage.getCountOfItems();
+    qanPage.verifyChangedCount(countBefore, countAfter);
+  }
+);
+
+xScenario('PMM-T126 - Verify user is able to Reset All filters @not-pr-pipeline', async (I, qanPage) => {
+  const service_name = 'ps_5.7_0.0.0.0_1';
+  const environmentName = 'ps-dev';
+  qanPage.waitForNewQANPageLoaded();
+  const countBefore = await qanPage.getCountOfItems();
+  qanPage.applyFilterNewQAN(service_name);
+  qanPage.applyFilterNewQAN(environmentName);
+  const countAfter = await qanPage.getCountOfItems();
+  await qanPage.verifyChangedCount(countBefore, countAfter);
+  I.click(qanPage.fields.resetAll);
+  I.waitForVisible(qanPage.fields.resetAll + ':disabled', 20);
+});
+
+xScenario(
```

Comment:
Let's defer this commit to a later time when the test is ready to be run.

Reply:
@atymchuk I don't understand, what do you mean. :) Could you explain it more to me? :)

Reply:
Sure. I mean we should not commit code that does nothing. Same is true for the tests: once we are ready to run these tests, only then we should commit those.
Right now they are disabled (which similar to commented out), so there is no chance to run them and verify them.

Reply:
Understand. But whole this task is about test cases for new QAN, which have not be deployed yet. So I don't know what should I do with this now. :)